validator/fixtures/datasets.json

```diff
@@ -171,5 +171,27 @@
 				15
 			]
 		}
+	},
+	{
+		"model": "validator.dataset",
+		"pk": 8,
+		"fields": {
+			"short_name": "ERA5",
+			"pretty_name": "ERA5",
+			"help_text": "",
+			"source_reference": "",
+			"citation": "",
+			"is_reference": false,
+			"versions": [
+				14,
+				15
+			],
+			"variables": [
+				11
+			],
+			"filters": [
+				1
```

Comment:
Together with a new ERA5 soil temperature var (stl1), we can filter for frozen soil